azkaban-common/src/test/java/azkaban/executor/ExecutionFlowDaoTest.java

```diff
@@ -762,6 +762,29 @@ public void testFlowStatusWithFetchRecentlyFinishedFlows() throws Exception {
     assertThat(finishedFlows2.get(0).getStatus()).isEqualTo(Status.FAILED);
   }
 
+  @Test
```

Comment:
Thanks for adding a test! A short description for the test would be useful.

Reply:
I thought the name was self explanatory, however, I will add some comments.